pkg/controller/stats/aggregator.go

```diff
@@ -369,3 +371,36 @@ func addUp(stats *statsv1alpha1.TrafficStats, inc *statsv1alpha1.TrafficStats) {
 	stats.Packets += inc.Packets
 	stats.Bytes += inc.Bytes
 }
+
+func addRulesUp(ruleStats *[]statsv1alpha1.RuleTrafficStats, ruleSumStats *statsv1alpha1.TrafficStats, inc map[string]statsv1alpha1.TrafficStats) {
```

Comment:
Slice is kind of pointer type, no need to use pointer to pass value.

Reply:
the function may append the slice `ruleStats `, so pointer type is needed

Reply:
I see. Golang normally does this by returning the updated slice. For instance: `ruleStats = append(ruleStats, ...)`

Reply:
For simplicity, I will keep current implementation for this function.

If I change this function to return the updated slice, I am afraid for coherence, the ruleSumStats should be also be passed as value and returned a new one. After applying `addRulesUp` function, we need to assign these returned values to curStats's `trafficstats` and `ruletrafficstats`. The code looks verbose in this way.